app/views/products/index.html.erb

```diff
@@ -16,11 +17,12 @@
             <%= link_to product_path(product.id) do %>
                 <article class="product-item">
                     <img class="product-page-image" src= <%= product.photo_url %> alt="Image of <%= product.name %>">
-                    <p><%= product.name %></p>
+                    <p><%= link_to product.name, product_path(product.id) %></p>
```

Comment:
So the way i wrote this makes it so the entire article is clickable! so we don't need to make the product name it's own link (unless you want to! then we should remove the link around the article)

Reply:
I missed the article being clickable! We should def keep it the way you had it!